common/lib/protocol-definitions/src/storage.ts

```diff
@@ -114,6 +114,11 @@ export interface ISnapshotTree {
      * Indicates that this tree is unreferenced. If this is not present, the tree is considered referenced.
      */
     unreferenced?: true;
+
+    /**
+     * Indicates that this tree is tombstoned. If this is not present, the tree is considered not tombstoned.
+     */
+    tombstoned?: true;
```

Comment:
Simply adding a flag here won't work. To add this, we need to establish a new protocol with the driver / server, send them this tombstone value in summary which they will have to send back in snapshot.

Reply:
Yeah, I added it here to see if we wanted to establish this protocol between the driver and server